### PR TITLE
Fixed issue with jquery-validator

### PR DESCRIPTION
**Pull request by tonmoy18**
Previously jquery validator was trying to add
validator on funderClass on every place.
This was generating an error. Added a check to
see if the field actually exists befor adding.

### CODE REVIEW

**Comment by tonmoy18** [comment]
LIBTD-252 on JIRA

File: app/assets/javascripts/datarepo/funder.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ $(".simple_form").validate({
 			error.insertAfter( element );
 	}
 });
-$(funderClass).rules("add", {datarepoFunder: ".*:.*"});
-
+if ($(funderClass).length > 0) {
```
Comment (rotated8) [inline app/assets/javascripts/datarepo/funder.js]: You can remove the parentheses around the entire statement.

Comment (tonmoy18) [inline app/assets/javascripts/datarepo/funder.js]: Isn't that a bad practice?

Comment (rotated8) [inline app/assets/javascripts/datarepo/funder.js]: I don't think so, we don't have multiple parts that are being combined, it's just one test. Sufia does this as `if $(class).length {`

Comment (tonmoy18) [inline app/assets/javascripts/datarepo/funder.js]: Oh, sorry, I thought you were talking about the curly braces.

**Comment by rotated8** [comment]
:+1:

**Pull request merged**